faker/faker.go

```diff
@@ -52,7 +53,13 @@ func (f faker) getFakedValue(a interface{}) (reflect.Value, error) {
 				}
 				val, err := f.getFakedValue(v.Field(i).Interface())
 				if err != nil {
-					fmt.Println(err)
+					if err == errEFaceNotAllowed { // skip empty interfaces
+						continue
+					}
+
+					if f.verbose {
+						fmt.Println(err)
```

Comment:
can we use logger instead of `fmt.Println` this way we wont need the verbose flag as well as it will but just configured by the parent logger